apps/browser/src/autofill/background/overlay.background.spec.ts

```diff
@@ -2899,6 +2899,124 @@ describe("OverlayBackground", () => {
         );
       });
     });
+    describe("handles menu position when input is focused", () => {
+      it("sets button and menu width and position when non-multi-input totp field is focused", async () => {
+        const subframe = {
+          top: 0,
+          left: 0,
+          url: "",
+          frameId: 0,
+        };
+
+        overlayBackground["focusedFieldData"] = createFocusedFieldDataMock({
+          focusedFieldRects: {
+            width: 49.328125,
+            height: 64,
+            top: 302.171875,
+            left: 1270.8125,
+          },
+        });
+
+        const buttonPostion = overlayBackground.getInlineMenuButtonPosition(subframe);
+        const menuPostion = overlayBackground.getInlineMenuListPosition(subframe);
```

Comment:
should allow keeping the methods as private (here and elsewhere)

```suggestion
        const buttonPostion = overlayBackground['getInlineMenuButtonPosition'](subframe);
        const menuPostion = overlayBackground['getInlineMenuListPosition'](subframe);
```